driver-moab/main/component_mod.F90

```diff
@@ -748,6 +748,7 @@ subroutine component_init_areacor_moab (comp, mbccid, mbcxid, seq_flds_c2x_fluxe
          lsize = comp(1)%mblsize
          allocate(areas (lsize, 3)) ! lsize is along grid; read mask too
          allocate(factors (lsize, 2))
+         factors = 1.0 ! initialize with 1.0 all factors; then maybe correct them
```

Comment:
Shouldn't it be factors(:,:) = 1.0? Or is the syntax in general the same (and independent of dimension or compiler family)?

Reply:
we are not consistent, we use both forms, but this one is shorter. 
We are consistent when we use set/get methods with these 2 dimensional arrays, iMOAB_SetDoubleTagStorage, iMOAB_GetDoubleTagStorage; 
`ierr = iMOAB_SetDoubleTagStorage( mbccid, tagname, arrsize , comp(1)%mbGridType, factors)`